src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -115,10 +129,11 @@ public void save(AddressBook addressBook) throws StorageOperationException {
      * Loads data from this storage file.
      *
      * @throws StorageOperationException if there were errors reading and/or converting data from file.
+     * @throws FileNotFoundException
      */
-    public AddressBook load() throws StorageOperationException {
+    public AddressBook load() throws StorageOperationException, FileNotFoundException {
```

Comment:
I noticed that you are throwing FileNotFoundException` in this method, though it is already handled within this method.